src/game.rs

```diff
@@ -0,0 +1,250 @@
+use macroquad::{
+    color::{BLACK, WHITE},
+    shapes::draw_rectangle,
+    window::{clear_background, next_frame},
+};
+
+/// Represents the state of the Game of Life.
+struct Game {
+    height: usize,
+    width: usize,
+    cells: Vec<bool>,
```

Comment:
Why don't you use a matrix instead?

Reply:
Agree, it'd be more readable with a Vec<Vec\<Bool> and indexing would be more intuitive.

Reply:
You're right, I'll change that